### DISCUSSION TITLE
Pole figure from Euler angle csv file

### DISCUSSION THREAD

**Discussion by makeabhishek** [mtex-toolbox/mtex#1010] [Ask Anything] [answered]
Hi,
I hope everyone is keeping well.

I am new to use MTEX. I have a csv file with five columns data obtained from EBSD, which include data in this format.

Width 2419
Height 1140
X Position       Y Position         Eul 1         Eul 2         Eul 3 

I want to plot pole figure from this Euler angle data. Kindly advice.

Regards
Abhishek
--

**Comment by ralfHielscher** [comment, marked as answer]
You can do
```matlab
cs = crystalSymmetry('yourSymmetryGroup')
ori = orientation.load('fileName','columnNames',{'x','y','Euler1','Euler2','Euler3'},cs)
plotPDF(ori,Miller(1,0,0,cs),'contourf')
```
Ralf.

**Comment by makeabhishek** [comment]
Thanks Ralf for your quick response. I am working on Titanium 64 alloy to
investigate the wave propagation in the metals. I know Titanium is a
hexagonal crystal symmetry with alpha and beta phases. I would like to take
your advice on the crystal symmetry type for TI64 alloy.
What should I choose for crystal symmetry?

Thanks
Abhishek

On Thu, May 6, 2021 at 1:09 PM Ralf Hielscher ***@***.***>
wrote:

> You can do
>
> cs = crystalSymmetry('yourSymmetryGroup')
> ori = orientation.load('fileName','columnNames',{'x','y','Euler1','Euler2','Euler3'},cs)
> plotPDF(ori,Miller(1,0,0,cs),'contourf')
>
> Ralf.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mtex-toolbox/mtex/discussions/1010#discussioncomment-702202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALNXRFCKI4KCFAXPHJU5WTTTMIQAVANCNFSM44GJCNAA>
> .
>


-- 
Thanks & Regards
Abhishek Saini


This email and any attached files are confidential and intended solely for
the intended recipient(s). If you are not the named recipient you should
not read, distribute copy or alter this email. Any views or opinions
expressed in this email are those of the author and do not represent those
of the organization. Warning: Although precautions have been taken to make
sure no viruses are present in this email, the organization cannot accept
responsibility for any loss or damage that arise from the use of this email
or attachments.
